sinfonier-web/app/utils/PageElement.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+public class PageElement {
+  private int pageNumber;
+  private String pageLinkText;
+  
+  public PageElement(int pageNumber, String pageLinkText) {
+    super();
```

Comment:
This is product of "Generate Constructor using Fields" template of Eclipse. Not necessary here...